src/components/tab-title/tab-title.e2e.ts

```diff
@@ -108,49 +377,72 @@ describe("calcite-tab-title", () => {
     });
 
     describe("scale property", () => {
-      it("should inherit small scale from tab-nav", async () => {
-        const page = await newE2EPage({
-          html: `<calcite-tab-nav scale="s">
-            <calcite-tab-title selected>Tab Title</calcite-tab-title>
-            <calcite-tab-title>Tab 2 Title</calcite-tab-title>
-          </calcite-tab-nav>`
-        });
-        const element = await page.find("calcite-tab-title");
-        const container = await page.find("calcite-tab-title >>> .container");
-        const containerStyles = await container.getComputedStyle();
-        expect(element).toEqualAttribute("scale", "s");
-        expect(containerStyles.fontSize).toEqual("12px");
-        expect(containerStyles.lineHeight).toEqual("16px"); // 1rem
+      let page: E2EPage;
+
+      beforeEach(async () => {
+        page = await newE2EPage();
       });
 
-      it("should inherit medium scale from tab-nav", async () => {
-        const page = await newE2EPage({
-          html: `<calcite-tab-nav scale="m">
-            <calcite-tab-title selected>Tab Title</calcite-tab-title>
-            <calcite-tab-title>Tab 2 Title</calcite-tab-title>
-          </calcite-tab-nav>`
-        });
-        const element = await page.find("calcite-tab-title");
-        const container = await page.find("calcite-tab-title >>> .container");
-        const containerStyles = await container.getComputedStyle();
-        expect(element).toEqualAttribute("scale", "m");
-        expect(containerStyles.fontSize).toEqual("14px");
-        expect(containerStyles.lineHeight).toEqual("16px"); // 1rem
+      it("should inherit default medium scale from tab-nav", async () => {
+        await page.setContent(
+          html`
+            <calcite-tab-nav scale="m">
+              <calcite-tab-title selected>Tab Title</calcite-tab-title>
+              <calcite-tab-title>Tab 2 Title</calcite-tab-title>
+            </calcite-tab-nav>
+          `
+        );
+
+        expect(await page.find("calcite-tab-nav")).toEqualAttribute("scale", "m");
+        expect(await page.find("calcite-tab-title")).toEqualAttribute("scale", "m");
+
+        const content = await page.find(`calcite-tab-title >>> .${CSS.content}`);
+        const contentStyles = await content.getComputedStyle();
+
+        expect(contentStyles.fontSize).toEqual("14px");
+        expect(contentStyles.lineHeight).toEqual("16px");
+      });
+
+      it("should inherit small scale from tab-nav", async () => {
+        await page.setContent(
+          html`
+            <calcite-tab-nav scale="s">
+              <calcite-tab-title selected>Tab Title</calcite-tab-title>
+              <calcite-tab-title>Tab 2 Title</calcite-tab-title>
+            </calcite-tab-nav>
+          `
+        );
+        await page.waitForChanges();
```

Comment:
I think you don't need to call `await page.waitForChanges();` after setting content.

Applies to similar calls.